pages/about-our-work/overview.md

```diff
@@ -4,21 +4,30 @@ layout: styleguide
 title: About our work
 category: About our work
 lead: We’re always looking for ways to improve the Standards. Here, you'll find the latest information about recent releases, our product roadmap, and our governance model.
+columns:
+  - title: New users
+    source: New Users
+  - title: Total users
+    source: Users
+  - title: Page views
+    source: Pageviews
 ---
-<div class="usa-grid-full">
-  <div class="usa-width-one-half">
-    <h3>Estimated People Exposed to Draft U.S Web Design Standards</h3>
-    <p>By accessing the data that is gathered via analytics.usa.gov, we have aggregated a few data points to showcase how many users of federal websites have been exposed to the Draft U.S. Web Design Standards. These numbers will be updated quarterly.</p>
-    <h3>Quarter 3 of 2016 </h3>
-    <h3>62,437,565 Total people exposed</h3>
-    <h4>54,646,374 First time exposed</h4>
-    <h4>478,526,298 Across Page Views</h4>
-  </div>
-  <div class="usa-width-one-half">
-    <h3>Latest Community Engagement Report via Github Repo</h3>
-    <p>Thanks to the community we have fostered during the development of the Draft U.S. Web Design Standards, we have gathered the following data points which highlights the activity of our code base. </p>
-    <h3>Quarter 3 of 2016</h3>
-    <h4>507 Code Commits</h4>
-    <h4>73 Forks</h4>
-  </div>
-</div>
+
+### Web analytics for [sites that use the Standards](#sites-that-use-the-standards)
+
+By accessing the Google Analytics data gathered for
+[analytics.usa.gov](https://analytics.usa.gov), we have aggregated a
+few data points to approximate how many users of federal websites
+have "experienced" the Draft U.S. Web Design Standards. We will
+update these numbers quarterly.
+
+{% include reports/analytics.html columns=page.columns %}
+
+#### Sites that use the standards
+
+### GitHub engagement
```

Comment:
@bradnunnally we still cool with removing Github info from reporting? I vaguely remember us talking about doing that at one point.